packages/dds/tree/src/test/util/utils.spec.ts

```diff
@@ -70,4 +71,23 @@ describe("Utils", () => {
 		assert.equal(objWithProperty.prop, 3);
 		assert.equal(count, 1);
 	});
+
+	describe("copyProperty", () => {
+		it("copies a known property", () => {
```

Comment:
Probably worth adding a test case where the property is explicitly present in the input, but is `undefined` (as opposed to being a missing optional property).

Reply:
Done!